pkg/sql/ttl/ttljob/ttljob.go

```diff
@@ -127,17 +248,49 @@ func (t rowLevelTTLResumer) Resume(ctx context.Context, execCtx interface{}) err
 		if ttl == nil {
 			return errors.Newf("unable to find TTL on table %s", desc.GetName())
 		}
+		ttlSettings = *ttl
 
 		ranges, err = kvclient.ScanMetaKVs(ctx, txn, desc.TableSpan(p.ExecCfg().Codec))
 		if err != nil {
 			return err
 		}
-		ttlSettings = *ttl
+
+		_, dbDesc, err := descs.GetImmutableDatabaseByID(
+			ctx,
+			txn,
+			desc.GetParentID(),
+			tree.CommonLookupFlags{
+				Required: true,
+			},
+		)
+		if err != nil {
+			return err
+		}
+		schemaDesc, err := descs.GetImmutableSchemaByID(
```

Comment:
agh i wish it were easier to make a table name without having to fetch these descriptors :/